### ISSUE TITLE
[Adaptive Cards - Input]: Talkback is conveying irrelevant information while navigating 'Input' screen using 'Talkback+Keyboard'.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#105]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)
Keyboard: Logitech K480

**Repro Steps:**
Launch the Adaptive cards app on Android and turn on the Talkback.
Navigate to 'Import' control and select it.
Navigate to 'Sample cards' and select it.
Navigate to available dropdown under 'Sample cards' and select it.
Navigate all the controls under dropdown and select 'Stock Update'.
Navigate to 'Visual' tab control and select it.
Navigate to 'Input' option controls under 'Visual'.
Navigate the elements present under 'Inputs' using keyboard and observe the issue.

**Actual Result:**
While navigating the 'Input' screen using Tab or Shift+tab key with Talkback, screen reader is conveying irrelevant infomration to the user. It is announcing all infomration related to screen reader user not for the keyboard users. It is announcing as **'Double tap to activate it.'**

**Expected Result:**
Talkback should not announce irrelevant information to the user while navigating using keyboard with Tab and Shift+tab key. When user navigates using keys then screen reader should announce the information as 'Press enter or spacebar key to activate it'.

**User Impact:**
It will impact the keyboard user and screen reader user as they will miss the important infomration present on the screen related to control. Also, if there is any shortcut present then also, they will miss the infomration about the shortcut keys.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/130f97dd-d658-4811-9c8a-438fd372700b

### Card JSON

```JSON
NA
```
